go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/couchbaselabs/rosmar
 go 1.19
 
 require (
-	github.com/couchbase/sg-bucket v0.0.0-20230912183916-dfc81a029d3b
+	github.com/couchbase/sg-bucket v0.0.0-20230917233712-d10f9c9a67bf
```

Comment:
I guess this will need another bump once https://github.com/couchbase/sg-bucket/pull/105 is merged - will hold off merging